### ISSUE TITLE
Incorrect documentation for the command to start a v2 app  

### ISSUE THREAD

**Issue by gparsloe** [ionic-team/ionic-site#1086]
**Incorrect documentation** on this page of the Current Framework Docs: 

https://ionicframework.com/docs/intro/installation/

The command given (see below) will produce a v1 app, not a v2 app (the start command defaults to downloading the v1 templates, not the v2): 

Once that’s done, create your first Ionic app:
$ ionic start cutePuppyPic 

**Suggested correction:**  Add '-v2' to correct it, as shown below: 

Once that’s done, create your first Ionic app:
$ ionic start cutePuppyPic  **-v2**

**Comment by danielsogl** [comment]
Hey @gparsloe,
you are using an old CLI version then. The command is correct.

**Comment by gparsloe** [comment]
Hey @danielsogl,
Thanks! I was on Ionic CLI 2.2.2. Once I upgraded to 2.2.3, it worked as per the tutorial. 

I'm wondering if it would help some people if the documentation reflected that a specific version is required. I found it very confusing that I was getting a v1 app when I followed the tutorial instructions (and didn't have the 2.2.3 version). 

Suggestion: 

Once that’s done, create your first Ionic app:
$ ionic start cutePuppyPic 

Note: If you are using an Ionic CLI version less than 2.2.3, you must add --v2 to the command: $ionic start cutePuppyPic --v2
